application/src/test/java/org/thingsboard/server/transport/coap/rpc/AbstractCoapServerSideRpcIntegrationTest.java

```diff
@@ -75,14 +76,23 @@ protected void processOneWayRpcTest(boolean protobuf) throws Exception {
         CoapTestCallback callbackCoap = new TestCoapCallbackForRPC(client, 1, true, protobuf);
 
         CoapObserveRelation observeRelation = client.getObserveRelation(callbackCoap);
-        callbackCoap.getLatch().await(3, TimeUnit.SECONDS);
+        String awaitAlias = "await One Way Rpc (client.getObserveRelation)";
+        await(awaitAlias)
+                .atMost(10, TimeUnit.SECONDS)
+                .until(() -> CoAP.ResponseCode.VALID.equals(callbackCoap.getResponseCode()) &&
+                        callbackCoap.getObserve() != null &&
+                        0 == callbackCoap.getObserve().intValue());
         validateCurrentStateNotification(callbackCoap);
-
-        CountDownLatch latch = new CountDownLatch(1);
+        int expectedObserveBeforeGpioRequestCnt = callbackCoap.getObserve().intValue() + 1;
```

Comment:
Hi @nickAS21, the same issue with naming as in the attributes tests. Please consider renaming variable to expectedObserveCountAfterGpioRequest

Reply:
please fix the naming of observing counters in other places as well.

Reply:
OK